src/app/app.component.ts

```diff
@@ -17,7 +17,8 @@ export class MyApp {
     platform.ready().then(() => {
       // Okay, so the platform is ready and our plugins are available.
       // Here you can do any higher level native things you might need.
-      statusBar.overlaysWebView(false);
+      statusBar.overlaysWebView(true);
```

Comment:
Have we checked that this doesn't conflict with other prototypes? I remember having to do this to get the nav looking ok..